roles/ceph-common/tasks/main.yml

```diff
@@ -93,10 +88,24 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
-- include: facts.yml
-- include: ./checks/check_socket.yml
+- name: get ceph version
```

Comment:
Hmm -- looks like it never would have passed since PR #737 (which was merged long before support for rbd-mirror). Apparently the installation of rbd-mirror never worked.